projects/storefrontapp-e2e-cypress/cypress/integration/regression/textfield-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,76 @@
+import * as cart from '../../helpers/cart';
+import * as productSearch from '../../helpers/product-search';
+import * as textfiledConfiguration from '../../helpers/textfield-configuration';
+
+const electronicsShop = 'electronics-spa';
+const testProduct = '1934793';
+const ENGRAVED_TEXT = 'Engraved Text';
+const HALLO = 'Hallo';
+
+context('Textfield Configuration', () => {
+  before(() => {
+    cy.visit('/');
```

Comment:
have to login first, right? Otherwise, it will not go to product search page.

Reply:
No, this is a different test for a different configurator type that is supposed to run in b2c. No login needed.